project/DistributionPackage.scala

```diff
@@ -377,6 +375,9 @@ object DistributionPackage {
       all.add("--disable-private-check")
     }
     pb.command(all)
+    workingDirectory
```

Comment:
CCing @AdRiley:
- I believe this change of workingDirectory breaks a lot of usual commands
- for example, as reported by Adam:
```
sbt:enso> runEngineDistribution --docs api --in-project distribution/lib/Standard/Tableau/0.0.0-dev
```
- was broken, in my opinion